plugins/package-managers/swiftpm/src/funTest/kotlin/SwiftPmFunTest.kt

```diff
@@ -63,6 +63,18 @@ class SwiftPmFunTest : WordSpec({
         }
     }
 
+    "Analyzing a lockfile with unsupported file format version 4" should {
+        "return the correct result" {
+            val definitionFile = getAssetFile("projects/synthetic/only-lockfile-v4/Package.resolved")
+            val expectedResultFile = getAssetFile("projects/synthetic/expected-output-only-lockfile-v4.yml")
+
+            val result = create(PROJECT_TYPE).resolveSingleProject(definitionFile)
+
+            result.withInvariantIssues().toYaml() should matchExpectedResult(expectedResultFile, definitionFile)
+        }
+    }
+
+
```

Comment:
Ping about the needless blank lines (also see below), in case you overlooked it.